.github/workflows/backport-fail-bot.yml

```diff
@@ -9,25 +9,39 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.issue.pull_request != null && contains(github.event.comment.body, 'To backport manually, run these commands in your terminal')
     steps:
+    - name: Fetch mapping file
+      id: fetch_mapping
+      uses: actions/github-script@v6
+      env:
+        ACCESS_TOKEN: ${{ secrets.PAT }}
+      with:
+        script: |
+          const url = 'https://raw.githubusercontent.com/Kong/github-slack-mapping/main/mapping.json';
+          const headers = {Authorization: `token ${process.env.ACCESS_TOKEN}`};
+          const response = await fetch(url, {headers});
+          const mapping = await response.json();
+          return mapping;
+
       - name: Generate Slack Payload
         id: generate-payload
         uses: actions/github-script@v6
+        env:
+          SLACK_CHANNEL: gateway-notifications
+          SLACK_MAPPING: ${{ steps.fetch_mapping.outputs.result }}
         with:
           script: |
+            const pr = ${{ github.event.issue.pull_request }};
+            const pr_url = pr.html_url;
             const slack_mapping = JSON.parse(process.env.SLACK_MAPPING);
-            const pr_url = "${{ github.event.issue.pull_request.html_url}}";
-            const pr_author_github_id = "${{ github.event.issue.user.login }}"
+            const pr_author_github_id = pr.user.login;
             const pr_author_slack_id = slack_mapping[pr_author_github_id];
-            const author = (pr_author_slack_id ? `<@${pr_author_slack_id}>` : pr_author_github_id);
+            const author = pr_author_slack_id ? `<@${pr_author_slack_id}>` : pr_author_github_id;
             const payload = {
-              text: `Backport failed in PR: ${pr_url}. Please check it ${author}.`,
+              text: `Hello ${author} , backport failed in PR: ${pr_url}.`,
```

Comment:
I like the old message better, can we keep it? :)

Reply:
If the message was too long, we can put the username in the very beginning for a better reminder?

Reply:
What about:

```
${pr_url} from ${author} failed to backport.
```

Reply:
@dndx updated and rebased.